src/components/pages/ManageCases/ManageCases.js

```diff
@@ -214,7 +214,8 @@ export default function ManageCases(props) {
                   {caseObj.credible ? 'True' : 'False'}
                 </p>
                 <p style={{ margin: 0 }}>
-                  Violence Experienced: {caseObj.type_of_violence_experienced}
+                  Violence Experienced:{' '}

```

Comment:
I believe that Violence Experienced is meant to be changed to Persecution Experienced. Using the verbiage across the board, even outside of variable names, seems to be a key goal to maintain conceptual consistency. This would be changed on line 217

Reply:
Good catch and I appreciate the inline comment! Revisions have been pushed.